lxd/cluster/recover.go

```diff
@@ -538,3 +539,87 @@ func unpackTarball(tarballPath string, destRoot string) error {
 
 	return nil
 }
+
+// createTarball creates tarball at tarballPath, rooted at rootDir and including
+// all files in walkDir except those paths found in excludeFiles.
+// walkDir and excludeFiles elements are relative to rootDir.
+func createTarball(tarballPath string, rootDir string, walkDir string, excludeFiles []string) error {
+	tarball, err := os.Create(tarballPath)
+	if err != nil {
+		return err
+	}
+
+	gzWriter := gzip.NewWriter(tarball)
+	tarWriter := tar.NewWriter(gzWriter)
+
+	filesys := os.DirFS(rootDir)
+
+	err = fs.WalkDir(filesys, walkDir, func(filepath string, stat fs.DirEntry, err error) error {
+		if err != nil {
+			logger.Warn("Failed to read file while creating tarball; skipping", logger.Ctx{"file": filepath, "err": err})
```

Comment:
why skipping here?

Reply:
I think this must have been left over from my debugging; removed it.